examples/light-switch-app/efr32/src/AppTask.cpp

```diff
@@ -179,8 +179,11 @@ CHIP_ERROR AppTask::Init()
     EFR32_LOG("APP: Done WiFi Init");
     /* We will init server when we get IP */
 #endif
+
+    chip::DeviceLayer::PlatformMgr().LockChipStack();
```

Comment:
Any reason not to use `StackLock` here with a scope around it?  Similar in other places.